code/modules/mob/living/carbon/human/life.dm

```diff
@@ -462,7 +453,7 @@
 			stat = CONSCIOUS
 
 		//Eyes
-		if(disabilities & BLIND)	//disabled-blind, doesn't get better on its own
+		if(disabilities & BLIND || stat)	//disabled-blind, doesn't get better on its own

```

Comment:
stat == 1 is unconscious, stat == 2 is dead. It's possible to recover from it by sleep, apparently?